docs/030_user-guide/120_customization.md

```diff
@@ -132,5 +132,46 @@ Below are the available configurations through `package.json`.
 | `alwaysIgnore`   | Conditions to always ignore            | `{namespaces: []}`  |
 | `includedFiles`  | For working with WebAssembly           | ["main.wasm", "wasm_exec.js"]   |
 | `env`            | Environment variables for the container| `{LOG_LEVEL: "warn"}`           |
+| `rbac`          | Custom RBAC rules                    | `{"rules": [{"apiGroups": ["<apiGroups>"], "resources": ["<resources>"], "verbs": ["<verbs>"]}]}` |
 
 These tables provide a comprehensive overview of the fields available for customization within the Helm overrides and the `package.json` file. Modify these according to your deployment requirements.
+
+### Example Custom RBAC Rules
+
+The following example demonstrates how to add custom RBAC rules to the Pepr module.
+
+```json
+{
+  "pepr": {
+    "rbac": {
+      "roles": [
```

Comment:
this should be role and clusterRole since we only have one of each. The plural is misleading